plugins/UsersManager/Controller.php

```diff
@@ -436,6 +443,26 @@ public function setIgnoreCookie()
         Piwik::redirectToModule('UsersManager', 'userSettings', array('token_auth' => false));
     }
 
+    public function activeInviteUser()
+    {
+        $userLogin = Common::getRequestVar('login');
+        $token = Common::getRequestVar('token');
```

Comment:
When using request variables we should try to always provide the expected type to avoid possible issues when using them later. This can be easily achieved using the correct parameters. Would be good to keep that in mind for future PRs as well.

```suggestion
        $userLogin = Common::getRequestVar('login', null, 'string');
        $token = Common::getRequestVar('token', null, 'string');
```